src/components/Stepper.tsx

```diff
@@ -0,0 +1,76 @@
+"use client";
+
+import React from "react";
+import { ArrowRightIcon } from "@heroicons/react/24/solid";
+import Button from "@/components/Button";
+import { cn } from "@/lib/utils";
+// TODO FOR LATER REFACTOR:
+//  - setting the width for the stepper: do we want to specify the width or base it off parent's width
+//  - variant colors for stepper button
+
+// chips or blocks
+type StepperStyle = "chips" | "icons";
+type StepperStatus = "completed" | "current" | "remaining";
+
+export interface SteppersItem {
+  icon?: string;
```

Comment:
The icon should be an icon, not a string. You can set the type as ReactNode, that seems to work